corehq/apps/reports/standard/monitoring.py

```diff
@@ -471,7 +471,7 @@ def all_users(self):
         return sorted(users, key=lambda u: u['username_in_report'])
 
     def paginate_list(self, data_list):
-        if self.pagination:
```

Comment:
This will always come back true unless there is not a request object: https://github.com/benrudolph/commcare-hq/blob/export-all-data-156169/corehq/apps/reports/generic.py#L792

I wasn't sure what the best way was for explicitly marking no pagination, but this seemed reasonable